recipes/hwm/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "hwm" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/hwm-{{ version }}.tar.gz
+  sha256: 5f09159edf703390f4d3110d4737b2f2c47754a8bf11c2b0993e67270773f8c8
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }}
+    - numpy <2
+    - scipy >=1.7.0
+    - scikit-learn >=1.2.0
+    - pandas >=1.3.0
+    - joblib >=1.2.0
```

Comment:
@earthai-tech I don't see joblib listed upstream. Is it really needed?

Reply:
Not really needed for this version. I put it here for future version.

Reply:
Ok. Got it! Thanks